### ISSUE TITLE
Terms and Privacy page does not export or import

### ISSUE THREAD

**Issue by valadas** [dnnsoftware/Dnn.Platform#2518]
## Description of bug
The new default pages for terms and privacy do not export or import

## Steps to reproduce
List the steps to reproduce the behavior:
1. Create a site with a custom terms and privacy pages
2. Set those custom pages in the site settings and confirm they work
3. Export the site
4. Import it elsewhere
5. Terms and Privacy pages are blank after the import

## Current result
Terms and privacy page are set to <None Specified>

## Expected result
Should be the pages in the export

## Error log
Nothing special

## Additional context
I do not use the export/import feature much so if this is by design, please close and ignore.

## Affected version
* [x] 9.3.0.130

**Comment by kurtwilbies** [comment]
If using SCRUM as the base, I would say 9.3.1. Now, code freeze and test and fix issues.

**Comment by mitchelsellers** [comment]
I think we should look at this now, no more 1/2 implemented features. Let’s see if Peter has time to review.

**Comment by valadas** [comment]
I agree we should tackle this now, since it does not show an obvious error, users may not notice the problem and run into wrong terms on imported site, etc. Can you assign this to 9.3.0 milestone @mitchelsellers

**Comment by valadas** [comment]
Correct me if I am wrong, but I think this is currently no way to import/export localized portal settings. This may be a bigger task than expected, we may need to push it to 9.4. But if I am wrong please point me to the right direction...

**Comment by valadas** [comment]
I just realized that my comment may be unclear. The special pages are stored in the PortalLocalization table as are Custom404TabId and Custom500TabId. I have found no existing way to export this as it is now, 
So this is not specific to the recent additions, even the portal logo and other localized settings are currently not supported by Import/Export. I think this should be removed from the 9.3.0 milestone for that reason and that a correct fix would be to add localized settings import/export in an upcoming release.
